### PR TITLE
[stable/zfs-localpv]: update charts to 1.8.0

### PR DESCRIPTION
**Pull request by shubham14bajpai**
Signed-off-by: shubham <shubham.bajpai@mayadata.io>

**What this PR does?**:

This PR updates the helm charts for 1.8.0 release

**Does this PR require any upgrade changes?**:

**If the changes in this PR are manually verified, list down the scenarios covered:**:

**Any additional information for your reviewer?** :
_Mention if this PR is part of any design or a continuation of previous PRs_

**Checklist:**
- [ ] Fixes #<issue number>
- [ ] PR Title follows the convention of  `<type>(<scope>): <subject>`
- [ ] Has the change log section been updated?
- [ ] Commit has unit tests
- [ ] Commit has integration tests
- [ ] (Optional) Are upgrade changes included in this PR? If not, mention the issue/PR to track:
- [ ] (Optional) If documentation changes are required, which issue on https://github.com/openebs/openebs-docs is used to track them:

### CODE REVIEW

File: deploy/helm/charts/crds/zfsnode.yaml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+              properties:
+                free:
+                  anyOf:
+                  - type: integer
+                  - type: string
+                  description: Free specifies the available capacity of zfs pool.
+                  pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
+                  x-kubernetes-int-or-string: true
+                name:
+                  description: Name of the zfs zfs pool.
```
Comment (pawanpraka1) [inline deploy/helm/charts/crds/zfsnode.yaml:64]: zfs zfs pool?

Comment (shubham14bajpai) [reply deploy/helm/charts/crds/zfsnode.yaml]: I think I forgot to run `make manifest` in the last PR so the comments on the CRDs are wrong.

Comment (akhilerm) [reply deploy/helm/charts/crds/zfsnode.yaml]: you need to run `make helm` also, to keep CRDs to deploy/helm directory.

**Review by akhilerm — COMMENTED** [review]
@shubham14bajpai Why are the files in `deploy/yamls` and `deploy/zfs-operator.yaml` updated in the charts PR?

**Comment by shubham14bajpai** [comment]
> @shubham14bajpai Why are the files in `deploy/yamls` and `deploy/zfs-operator.yaml` updated in the charts PR?

Some of the CRD comments were addressed in the code but the yaml was missed in a previous PR. This just fixes those comments.

**Review by akhilerm — APPROVED** [review]
LGTM

**Review by pawanpraka1 — APPROVED** [review]
looks good.

**Pull request merged**